smk-edit/static/components/edit-item-details-layer-esri-dynamic.js

```diff
@@ -4,6 +4,47 @@ export default importComponents( [
 ] ).then( function () {
     return vueComponent( import.meta.url, {
         props: [ 'itemId' ],
+        computed: {
+            hasLabels: function() {
+                const layer = this.$store.getters.configLayer( this.itemId );
+                if (layer.dynamicLayers) {
+                    for (const dynamicLayer of layer.dynamicLayers) {
+                        let dynamicJson = JSON.parse(dynamicLayer);
+                        if(dynamicJson.drawingInfo.labelingInfo) {
```

Comment:
This the `dynamicJson` doesn't have a `drawingInfo` property, accessing `labelingInfo` will throw an exception. You can update this line to `if(dynamicJson.drawingInfo && dynamicJson.drawingInfo.labelingInfo) {`

Reply:
Done, here and below